esp8266-fastled-webserver/data/index.htm

```diff
@@ -174,6 +174,135 @@ <h4 class="panel-title">
       </div>
     </div>
 
+    <div id="utcOffsetIndexTemplate" class="form-group">
+      <label class="col-sm-2 control-label"></label>
+      <div class="col-sm-8">
+        <select class="form-control">
+          <option value="0">-12:00</option>
```

Comment:
As one alternative, the `value` settings would be in minutes:

Thus, -12:00 (currently index 0) would be `-720`, -11:45 would be `-705`, ... -8:00 would be -480, etc.